generate-new/src/Render/Element/Write.hs

```diff
@@ -283,8 +283,17 @@ renderModule out boot getDoc findModule findLocalModule (Segment modName unsorte
         let t = layoutDoc (d <> line <> line)
         if getAll (reCanFormat e)
           then liftIO (parsePrintModule brittanyConfig t) >>= \case
-            Left  _ -> error "Fail"
-            Right f -> pure f
+            Left  err -> error $ "Fail: Brittany failed to handle module:\n" <>
+                                    T.intercalate "\n" (fmap (T.pack . showBError) err)
+                                    <> "\n\n\n" <> t
+              where showBError = \case
```

Comment:
Yeah, I bumped in to this the other day, crazy that it doesn't have a show instance

Reply:
Yeah, what???